signaler.py

```diff
@@ -6,34 +6,28 @@
 from time import time, sleep
 
 def invader_detect():
-  """Takes a screen and looks for red clumps"""
-  image = screenshot()
-  red_count = 0
-  pink_count = 0
-  total = 0
+  """Takes a screenshot and looks for red clumps"""
+  image = screenshot(); image.save('temp.png')
+  red_count, pink_count = 0, 0
+  total = 0 # not used
+
   for pixel in image.getdata():
-    total += 1
-    if pixel[0] > 160 and pixel[1] < 55 and pixel[2] < 60:
-      red_count += 1
+    total += 1 # not used
+    if pixel[0] > 100 and pixel[1] < 50:
+      if pixel[2] < 50: red_count += 1
```

Comment:
The magic numbers here were tuned by hand by trial and error. Collapsing red & green checks for both might throw off detection.